dataprovider/src/main/java/cm/aptoide/pt/dataprovider/ws/v2/aptwords/GetAdsRequest.java

```diff
@@ -58,6 +59,7 @@ private GetAdsRequest(String clientUniqueId, boolean googlePlayServicesAvailable
     this.connectivityManager = connectivityManager;
     this.resources = resources;
     this.versionCodeProvider = versionCodeProvider;
+    this.groupPackageName = null;
```

Comment:
Is this necessary? We are not injecting the variable.

Reply:
fixed